server/routes/event.py

```diff
@@ -199,8 +201,13 @@ def event_node(dcid=DEFAULT_EVENT_DCID):
       subject_config.metadata.contained_place_types.update(
           DEFAULT_CONTAINED_PLACE_TYPES)
 
+    place_type = None
+    for pt in place_metadata.place_types:
+      if pt in DEFAULT_CONTAINED_PLACE_TYPES:
```

Comment:
since the contained_place_types in the config can either be `DEFAULT_CONTAINED_PLACE_TYPES` OR `place_metadata.contained_place_types_override`, shouldn't we use whichever we set to the config? Also in line 210

Also for disasters.py and sustainability.py

Reply:
yes - good point! done.